Sources/SwiftSgml/Tag.swift

```diff
@@ -98,3 +94,11 @@ open class Tag {
         return self
     }
 }
+
```

Comment:
This extension makes generating a `Node` name from a class name very compact. 

Easy to write and read.

Especially in all the HTML tag classes.